src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+2.3.12
```

Comment:
can you back out this change?

Reply:
Thanks for pointing out. Reverted this change